src/panels/PatientControlPanel.scss

```diff
@@ -0,0 +1,66 @@
+@import '../styles/variables';
+
+.patient-control-panel {
+    width: 100vw;
+    position: relative;
+    left: 50%;
+    right: 50%;
+    margin-left: -50vw;
+    margin-right: -50vw;
+
+    &, .FullApp-content {
+        background-color: $background;
+    }
+
+    .logo-accompaniment {
+        display: inline-block;
+        margin-bottom: 15px;
+
+        .title {
+            display: block;
+            color: $shr-context-dark;
+            font-size: 1.4rem;
+            font-weight: 600;
+            vertical-align: 0.5em;
+            padding-left: 5px;
+        }
+
+        .login {
+            display: block;
+            font-size: 0.8rem;
+            margin-left: 0.4rem;
+        }
+    }
+}
+
+.vertical-divider {
+    border-left: 1px solid $line-gray;
+    height: 40px
+}
+
+.search-icon {
+    padding-right: 5px;
+}
+
+/*
```

Comment:
Looks good on the whole! Can we just get rid of these commented out lines! Other than that I'm all good 👍

Reply:
Yeah no problem. Those were commented out in the original CSS file so I wasn't sure if they needed to stay or not.